linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/ClassEmitter.scala

```diff
@@ -1075,12 +1075,10 @@ private[emitter] final class ClassEmitter(jsGen: JSGen) {
 
   def genTopLevelJSClassExportDef(cd: LinkedClass,
       tree: TopLevelJSClassExportDef): js.Tree = {
```

Comment:
I think you can inline this method above.

Reply:
Done.